docs/assets/images/glue-icons.svg

```diff
@@ -0,0 +1,42 @@
+<svg xmlns="http://www.w3.org/2000/svg">
```

Comment:
Why are including glue icons?

Reply:
They are standard icons that we should use if we have use cases that match them.